content/en/providers/check-in/sp/_index.md

```diff
@@ -543,6 +545,22 @@ Example response:
 }
 ```
 
+###### Refresh Request with PKCE
+
+To combine the refresh token grant type with PKCE you need to make the following
+request:
+
+```shell
+curl -X POST "${TOKEN_ENDPOINT}" \
+-d "client_id=${CLIENT_ID}" \
+-d "grant_type=refresh_token" \
+-d "refresh_token=${REFRESH_TOKEN}" \
+-d "scope=openid%20email%20profile" | python -m json.tool;
```

Comment:
```suggestion
  -d "client_id=${CLIENT_ID}" \
  -d "grant_type=refresh_token" \
  -d "refresh_token=${REFRESH_TOKEN}" \
  -d "scope=openid%20email%20profile" | python -m json.tool;
```

Reply:
Please do the same for other code excerpts

Reply:
Resolved by commit [Add indent to curl data parameter](https://github.com/EGI-Federation/documentation/pull/332/commits/d4c45e1af19edd9a4d4cdaa64e897a218160bc87)